docs/source/user_manual/search_space.rst

```diff
@@ -27,10 +27,61 @@ KBMOD provides the ability to expand or contract the range of starting pixels us
 1. Adding a fixed sized buffer around the edge of the images. This is done using the ``x_pixel_buffer`` and ``y_pixel_buffer`` parameters to set separate buffers for the x and y dimensions. If ``x_pixel_buffer = 5`` the code will add 5 pixels to both sides of the image, using starting pixels from [-5, w + 5).
 2. Specifying absolute pixel boundaries. This is done using the ``x_pixel_bounds`` and ``y_pixel_bounds`` parameters. For example you can *reduce* the size of the search by using ``x_pixel_bounds = [10, 100]``, which will search [10, 100) regardless of the image's width. Similarly, you can increase the region of the search by setting the bounds outside the image's area.
 
-Velocity Grid
--------------
+Choosing Velocities
+-------------------
+
+Perhaps the most complex aspect of the KBMOD algorithm is how it defines the grid of search velocities. KBMOD allows you to define custom search strategies to best match the data. These include:
+* ``SingleVelocitySearch`` - A single predefined x and y velocity
+* ``VelocityGridSearch`` - An evenly spaced grid of x and y velocities
+* ``KBMODV1SearchConfig`` - An evenly spaced grid of velocity magnitudes and angles (this was the only option in v1.1 and before)
+* ``RandomVelocitySearch`` - Randomly sampled x and y velocities
+Additional search strategies can be defined by overriding the ``TrajectoryGenerator`` class in trajectory_generator.py.
+
+The search is selected and configured by a single ``generator_config`` parameter that takes a dictionary. The dictionary must contain a ``name`` entry that matches one of the options above. For example to search a single trajectory with a given ``vx = 1.0`` and ``vy = -2.0``, we would use::
+
+    generator_config = { "name": "SingleVelocitySearch", "vx": 1.0, "vy": -2.0 }
+
+If no generator_config is provided, then KBMOD uses the ``KBMODV1SearchConfig`` search strategy and pulls the configuration parameters from the top level. In this case if ``average_angle`` is not specified, it is estimated from the image's WCS.
```

Comment:
"In this case if ``average_angle`` is not specified, it is estimated from the image's WCS."

Is this actually the case? From below it looks like we raise a ValueError instead. Sorry if I got confused here, but hoping this can be corrected clarified